### PR TITLE
finite_elasticity and bioelectric_finite_elasticity

### PR DESCRIPTION
**Pull request by el-tomo**

### CODE REVIEW

File: src/monodomain_equations_routines.f90 [hunk 1]
```diff
@@ -1278,15 +1278,15 @@ SUBROUTINE MONODOMAIN_PRE_SOLVE(CONTROL_LOOP,SOLVER,ERR,ERROR,*)
     IF(ASSOCIATED(CONTROL_LOOP%PROBLEM)) THEN
```
Comment (chrispbradley) [inline src/monodomain_equations_routines.f90]: This file isn't really used any more.

Comment (el-tomo) [inline src/monodomain_equations_routines.f90]: what do you mean? Which file isn't used any more?

Comment (chrispbradley) [inline src/monodomain_equations_routines.f90]: monodomain_equations_routines.f90. This was originally put in by Sander Land and isn't in the correct form. The mono and bi domain equations are now in biodomain_equations_routines.f90. Add new code to this file as this one will likely be removed one rainy day.

Comment (el-tomo) [inline src/monodomain_equations_routines.f90]: All right! I just checked one of my examples and it seems that none of the subroutines in monodomain_equation_routines.f90 is called. No idea why I was looking at that file...

Comment (chrispbradley) [inline src/monodomain_equations_routines.f90]: OK, It won't matter if your change is added to the defunct file.

**Pull request merged**